### PR TITLE
Create CONTRIBUTING and CODE_OF_CONDUCT

### PR DESCRIPTION
**Pull request by harleo**
Add contributing and code of conduct file for contributors

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+### How to contribute
+Follow these simple steps:
+1. Have a look at the issue page of the repository [here](https://github.com/ifactory-solutions/awesome-react-app/issues).
+2. Choose an issue you would like to resolve/contribute.
+3. Leave a comment in the issue, stating that you are working on it.
+4. Fork the repository [here](https://github.com/ifactory-solutions/awesome-react-app/fork_select).
+5. Start making the changes to your forked repository.
+6. If you think you're done, submit a pull request (For help, click [here](https://help.github.com/articles/creating-a-pull-request/)).
+
+7. (Optional step) Go to the issue page which you resolved and leave a comment referencing the pull request.
```
Comment (italopessoa) [inline CONTRIBUTING.md]: You could also add a link to help those who don't know how to reference issues 😉

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+### Introduction
+First of all, thank you for considering to contribute! 🙏
+
+Everybody can help make this project a little more awesome, doesn't matter if it's a big or small contribution.
+Please make sure you follow the code of conduct, which is located in the repository master directory, in order for everyone to have a peaceful and progressive experience.
+Find out below, which steps are needed to help out exactly.
+
+### How to contribute
+Follow these simple steps:
+1. Have a look at the issue page of the repository [here](https://github.com/ifactory-solutions/awesome-react-app/issues).
```
Comment (italopessoa) [inline CONTRIBUTING.md]: Have a look at the [issue page](https://github.com/ifactory-solutions/awesome-react-app/issues) page of the repository 
Looks cleaner, what do you think?

**Review by italopessoa — CHANGES_REQUESTED** [review]

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at hi@harleo.de. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```
Comment (italopessoa) [inline CODE_OF_CONDUCT.md:37]: _hi@harleo.de_
@jvcjunior @jouderianjr do we have any other communication channel besides gitter?

**Review by italopessoa — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 2 above)
Comment (harleo) [reply CONTRIBUTING.md]: Definitely.

File: CONTRIBUTING.md (hunk 1 above)
Comment (harleo) [reply CONTRIBUTING.md]: I'll add that.

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (harleo) [reply CODE_OF_CONDUCT.md:37]: Oops, just noticed I still have my own email there, sorry about that.

**Comment by italopessoa** [comment]
Hey @harleo. Could you please change your branches **origin** and **base** to develop?

so it will be like
> commits into ifactory-solutions:develop from harleo:{any branch based on develop}

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (italopessoa) [reply CODE_OF_CONDUCT.md:37]: No problem. We're still deciding what's going to be used😸

**Review by harleo — COMMENTED** [review]
Should be good now regarding the contributing.md
Code of conduct needs to be edited the way you guys intend to use the report feature.
Also changed the base branch to develop.

**Comment by italopessoa** [comment]
Hey @harleo, could you please create a new PR, using develop as the origin branch?
We can't approve this PR, since it has some conflicts due to the fact you are using the master branch.
We only use develop branch to PRs, we're working to make this clear to new contributors.

**Comment by jvcjunior** [comment]
Hey @harleo I'm sorry about the branch mess.. haha.. we are starting this project and we don't have a lot of experience. We are going to merge everything to master branch, and now on accept PR to our master branch as well. Folowwing this guy here: https://gist.github.com/Chaser324/ce0505fbed06b947d962

Maybe we could talk here: 
https://gitter.im/ifactory-open-source/awesome-react-app

Thanks for your patience and efforts!  ❤️

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -13,6 +13,7 @@ Follow these simple steps:
 2. Choose an issue you would like to resolve/contribute.
 3. Leave a comment in the issue, stating that you are working on it.
 4. Fork the repository [here](https://github.com/ifactory-solutions/awesome-react-app/fork_select).
+5. Switch to the 'develop' branch of the repository.
```
Comment (italopessoa) [inline CONTRIBUTING.md]: Could you please change to `master`? Sorry about the mess 😅

**Review by italopessoa — CHANGES_REQUESTED** [review]

**Comment by harleo** [comment]
All good guys, no worries.
Do you want me to do another PR containing the two files later to the develop branch?
Or are you going to approve this one?

PS: Don't forget about the contact error in the code of conduct file!

**Review by italopessoa — APPROVED** [review]

**Review by paulonotz0r — CHANGES_REQUESTED** [review]
Hey, @harleo! Following what @italopessoa said, should be better to create a new branch based on `develop` (but I think we'll work based on master) with name like `docs/contributing_and_conduct_code` with commits following [Commit Message Convention](https://conventionalcommits.org/), like 
```
docs(contribute): Add CONTRIBUTING.md file

Add this file to guide how to get started and contribute
to this project.

This resolves (issue number)
```

**Pull request closed without merging**